h/accounts/views.py

```diff
@@ -482,7 +482,6 @@ def profile(self):
         model = {}
         if userid:
             model["email"] = User.get_by_userid(request.domain, userid).email
-        if request.feature('notification'):
```

Comment:
This isn't strictly a no-op change, but the original code looks wrong to me. There's an `if userid:` check two lines earlier which the subscriptions lookup blithely ignores.